tests/ui/conftest.py

```diff
@@ -14,31 +24,103 @@ def capabilities(capabilities):
     capabilities['marionette'] = True
     return capabilities
 
+
 @pytest.fixture
 def fxa_account(base_url):
-    url = DEV_URL if 'dev' in base_url else PROD_URL
+    url = DEV_URL if 'dev' or 'localhost' in base_url else PROD_URL
     return FxATestAccount(url)
 
 
-@pytest.fixture(scope='session')
-def jwt_issuer(base_url, variables):
+@pytest.fixture
+def jwt_issuer(base_url, json_file):
     try:
-        hostname = [urlparse.urlsplit(base_url).hostname]
-        return variables['api'][hostname]['jwt_issuer']
+        hostname = urlparse.urlsplit(base_url).hostname
+        return json_file['api'][hostname]['jwt_issuer']
     except KeyError:
         return os.getenv('JWT_ISSUER')
 
 
-@pytest.fixture(scope='session')
-def jwt_secret(base_url, variables):
+@pytest.fixture
+def jwt_secret(base_url, json_file):
     try:
-        hostname = [urlparse.urlsplit(base_url).hostname]
-        return variables['api'][hostname]['jwt_secret']
+        hostname = urlparse.urlsplit(base_url).hostname
+        return json_file['api'][hostname]['jwt_secret']
     except KeyError:
         return os.getenv('JWT_SECRET')
 
 
 @pytest.fixture
+def initial_data(transactional_db, live_server, base_url):
+    call_command('generate_addons', 10, app='firefox')
```

Comment:
Right, but how are you preventing it from executing when running against dev, for instance?